finality-provider/cmd/fpd/daemon/commit_pr.go

```diff
@@ -0,0 +1,79 @@
+package daemon
+
+import (
+	"fmt"
+	"path/filepath"
+	"strconv"
+
+	bbntypes "github.com/babylonlabs-io/babylon/types"
+	fpcmd "github.com/babylonlabs-io/finality-provider/finality-provider/cmd"
+	fpcfg "github.com/babylonlabs-io/finality-provider/finality-provider/config"
+	"github.com/babylonlabs-io/finality-provider/finality-provider/service"
+	"github.com/babylonlabs-io/finality-provider/log"
+	"github.com/babylonlabs-io/finality-provider/util"
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/spf13/cobra"
+)
+
+// CommandCommitPubRand returns the commit-pubrand command by connecting to the fpd daemon.
+func CommandCommitPubRand() *cobra.Command {
+	var cmd = &cobra.Command{
+		// TODO: add --start-height as an optional flag
```

Comment:
I start to feel that adding `--start-height` as an option is important because of BTC-timestamping. A commit will be become effective after the committed epoch is finalized, which would take `100` BTC block time. Therefore, if the commit starts from the last committed height, which is behind, then all the randomness for the consumer blocks generated during `100` BTC block will be wasted